Core/HLE/sceUsbCam.cpp

```diff
@@ -186,7 +185,12 @@ void Register_sceUsbCam()
 
 void Camera::pushCameraImage(long long length, unsigned char* image) {
 	std::lock_guard<std::mutex> lock(videoBufferMutex);
-	videoBufferLength = length;
 	memset (videoBuffer, 0, sizeof(videoBuffer));
-	memcpy (videoBuffer, image, length);
+	if (length > sizeof(videoBuffer)) {
+		videoBufferLength = 0;
+		ERROR_LOG(HLE, "pushCameraImage: length error");
+	} else {
+		videoBufferLength = length;
+		memcpy (videoBuffer, image, length);
```

Comment:
Minor style issue: We don't have spaces after the function names in calls. Feel free to fix other instances of this that are close to your changes, too, like the memset above.